requirements.yml

```diff
@@ -20,7 +20,7 @@
 #- src: galaxyproject.slurm - Waiting for a new release that sets up clustering for dbd properly
```

Comment:
Is there an updated slurm role too?

Reply:
yes, 0.1.3 released 2 months ago but it looks like the only new commit is this one https://github.com/galaxyproject/ansible-slurm/commit/a3104b029f02c161cc983f0f30551e8d516d3377